metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/utils/EntityTypeUtils.java

```diff
@@ -0,0 +1,83 @@
+package eu.europeana.enrichment.utils;
+
+import eu.europeana.corelib.solr.entity.AbstractEdmEntityImpl;
+import eu.europeana.corelib.solr.entity.AgentImpl;
+import eu.europeana.corelib.solr.entity.ConceptImpl;
+import eu.europeana.corelib.solr.entity.OrganizationImpl;
+import eu.europeana.corelib.solr.entity.PlaceImpl;
+import eu.europeana.corelib.solr.entity.TimespanImpl;
+import eu.europeana.enrichment.api.internal.AgentTermList;
+import eu.europeana.enrichment.api.internal.ConceptTermList;
+import eu.europeana.enrichment.api.internal.MongoTermList;
+import eu.europeana.enrichment.api.internal.OrganizationTermList;
+import eu.europeana.enrichment.api.internal.PlaceTermList;
+import eu.europeana.enrichment.api.internal.TimespanTermList;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Simon Tzanakis
+ * @since 2020-07-17
+ */
+public class EntityTypeUtils {
```

Comment:
IDE: Add a private constructor to hide the implicit public one

Reply:
Done